### ISSUE TITLE
Prepare bash script with installer of dependent packages

### ISSUE THREAD

**Issue by grzanka** [grzanka/cookiecutter-pip-docker-versioneer#20]
numpy and scipy should be installed from system repositories, not using pip

**Comment by ant6** [comment]
What about other (probably) required packages e.g. fortran?
According to my previous tests with travis those are required:
```
addons:
  apt:
    packages:
    - libblas-dev
    - liblapack-dev
    - gfortran
```

**Comment by grzanka** [comment]
Well, can be done like this.
I am just not sure whether we should use addons or not.
If you put things in bash script then you can run your test suite also locally.

Imagine tests in docker: you grab fresh image of fedora. First you install git, then you grab the code from github. Then you run bash script to install dependent packages like blas/gfortran.

What is problematic on travis is that if you use tox, you are kind of forced to use pip version of numpy and scipy (there are two layers of virtualenv: one created by "language:python" in travis and another created by tox). Its hard to configure tox in such way that through these two layers it'll see numpy/scipy installed in the system.
On the other hand installation of scipy via pip means compilation of fortran code which takes incredibly long (>10min). I am trying to avoid this problem by using pip cache on travis.
It is quite cumbersome setting.

PS. I am working on that, so in order not do duplicate work, you can focus on sth else.

**Comment by grzanka** [comment]
Should be closed with PR #25

**Comment by grzanka** [comment]
With #28 and #29 I consider it closed
